Quests/Story EP6 names/Chapter 1/quest_00301440.csv

```diff
@@ -1,4 +1,4 @@
-Title#0,"""Assault of the «Phantom»"""
+Title#0,"""Assault of the \""Phantom\"""""
```

Comment:
Revert that, it was stylized like that on purpose.

Reply:
Reverted.